stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Flag.kt

```diff
@@ -20,7 +20,7 @@ public data class Flag(
     @SerializedName("reviewed_at")
     val reviewedAt: Date,
     @SerializedName("reviewed_by")
-    val approvedAt: Date,
+    val approvedAt: Date?,
```

Comment:
It turned out that `approvedAt` should be nullable. I have no idea how does it work with Gson because the property is always null although it's non-nullable